client/my-sites/plans/ecommerce-trial/index.tsx

```diff
@@ -31,6 +33,32 @@ const ECommerceTrialPlansPage = ( props: ECommerceTrialPlansPageProps ) => {
 		return getWooExpressMediumFeatureSets( { translate, interval } );
 	}, [ translate, interval ] );
 
+	const performanceOnlyOption = (
+		<ECommercePlanFeatures
+			interval={ interval }
+			monthlyControlProps={ { path: plansLink( '/plans', siteSlug, 'monthly', true ) } }
+			planFeatureSets={ wooExpressMediumPlanFeatureSets }
+			siteSlug={ siteSlug }
+			triggerTracksEvent={ triggerTracksEvent }
+			yearlyControlProps={ { path: plansLink( '/plans', siteSlug, 'yearly', true ) } }
+		/>
+	);
+
+	const tableProps = {
+		plans: [ PLAN_FREE, PLAN_WOOEXPRESS_MEDIUM ],
+		hidePlansFeatureComparison: true,
+	};
+
+	const performanceAndEssentialOption = (
+		<div className="is-2023-pricing-grid">
+			<AsyncLoad require="calypso/my-sites/plan-features-2023-grid" { ...tableProps } />
+		</div>
+	);
+
+	const upgradeOptions = isEnabled( 'plans/wooexpress-small' )
+		? performanceAndEssentialOption
+		: performanceOnlyOption;
```

Comment:
Nit: I think the naming here could be more self-descriptive, but that's not blocking. In particular, I think it would be worth changing the variable names something like the following:
 * `performanceOnlyOption` => `performanceOnlyFeatures`
 * `tableProps` => `plansTableProps`
 * `performanceAndEssentialOption` => `multiPlanFeatures`
 * `upgradeOptions` => `availablePlanFeatures`